src/util.rs

```diff
@@ -253,29 +253,27 @@ pub fn get_latest_runtime_version(runtime: &str) -> Result<String, String> {
     }
 }
 
-#[cfg(feature = "update-notifications")]
+#[derive(Debug, Copy, Clone, PartialEq, PartialOrd, Eq, Ord)]
+pub enum VersionComparison {
```

Comment:
I'm not opposed to having a custom enum, but this also just could be [std::cmp::Ordering](https://doc.rust-lang.org/stable/std/cmp/enum.Ordering.html).

Reply:
I would stick with this because it emphasizes which direction the comparison goes: `NewIsGreater` vs `Greater`.